src/Sylius/Bundle/ApiBundle/Resources/config/serialization/OrderItem.xml

```diff
@@ -17,6 +17,7 @@
 >
     <class name="Sylius\Component\Core\Model\OrderItem">
         <attribute name="id">
+            <group>account:order:read</group>
```

Comment:
I would suggest following strategy for serialization groups:
```suggestion
            <group>shop:order:account:read</group>
```
Which represent following: `context:resource:custom_action:operation`